### PR TITLE
Em/iteration3 sleep class

### PR DESCRIPTION
**Pull request by edmdc**
#### What's this PR do?
This is the beginning of the tests for the Sleep class. There will be more tests coming soon, to check for sad paths. 

#### Where should the reviewer start?

Glance over the sleep-test.js file and see what the test are asking for. 

#### How should this be manually tested?
npm test

### CODE REVIEW

File: src/scripts.js [hunk 1]
```diff
@@ -43,18 +50,42 @@ const displayUserFriends = () => {
   user.friends.forEach((friend) => {
     foundFriend = userRepository.findUser(friend)
     friendsSection.insertAdjacentHTML('beforeend',
-    `<p>Name: ${foundFriend.name}</p>
-     <p>Email: ${foundFriend.email}</p>
-     <p>Saily Step Goal: ${foundFriend.dailyStepGoal}</p>
-     <p>Stride Length: ${foundFriend.strideLength}</p>`)
+    `<div class="friend">
+      <p>Name: ${foundFriend.name}</p>
```
Comment (jeremiahblackol) [inline src/scripts.js:55]: not sure how this change adds to the project.

File: src/scripts.js [hunk 2]
```diff
@@ -2,16 +2,15 @@
 console.log("Hello World");
 
 let user;
-let userRepository;
+const userRepository = new UserRepository(userData);
```
Comment (jeremiahblackol) [inline src/scripts.js:5]: looks like its doing the same thing to me.

File: src/scripts.js [hunk 3]
```diff
@@ -43,18 +50,42 @@ const displayUserFriends = () => {
...
+  userDayHydrationWidget.innerHTML = 
+    `<h5> Daily Hydration: </h5>
+    <h4> ${userHydration.flOzConsumed} fl. Oz </h4>`
+};
 
+const displayWeekHydration = (userHydration) => {
+  const userWeekdayHydration = document.querySelector('.weekday-hydration-display');
+  const userWeekHydrationData = userHydration.weekHydrationData
+  userWeekHydrationData.forEach( day => {
+    let dayDisplay = new Date(day.date);
```
Comment (jeremiahblackol) [inline src/scripts.js:85]: This is cool!

**Review by jeremiahblackol — APPROVED** [review]

File: src/scripts.js (hunk 1 above)
Comment (edmdc) [reply src/scripts.js:55]: I didn't find anything about displaying friends' email. I took it out to make for cleaner appearance. Similarly, putting the paragraph text inside a div makes it easier for me to work in CSS.

Comment (edmdc) [reply src/scripts.js:55]: Makes it easier to work with in CSS.

**Pull request merged**